lib/rest.js

```diff
@@ -218,7 +218,7 @@ module.exports = function (app, config) {
 
         // Providing an id overwrites giving a query in the URL
         if (req.params.id) {
-            query = {'_id': new BSON.ObjectID(req.params.id)};
+            query = {'_id': req.params.id};
```

Comment:
This looks like it's going to break normal queries. Has it been tested for that?